### PR TITLE
[Improve][Zeta][Config] Add advanced-network-configuration in default config file

### PR DESCRIPTION
**Pull request by liugddx**
## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/incubator-seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by liugddx** [comment]
I have been tested,PTAL thanks. @TyrantLucifer

**Comment by TyrantLucifer** [comment]
> I have been tested,PTAL thanks. @TyrantLucifer

Give me some time. cc @john8628 Could you please help us test it?

**Comment by john8628** [comment]
> > I have been tested,PTAL thanks. @TyrantLucifer
> 
> Give me some time. cc @john8628 Could you please help us test it?

OK ,I will test it recently

**Review by john8628 — DISMISSED** [review]
good job; It works well for me.

**Comment by liugddx** [comment]
> good job; It works well for me.

Thank you to help test .

@TyrantLucifer @Hisoka-X PTAL , thanks.

**Comment by TyrantLucifer** [comment]
I didn't see any it cases of zeta be exceuted in CI/CD, how to trigger it manually? cc @EricJoy2048 @Hisoka-X

**Comment by Hisoka-X** [comment]
> I didn't see any it cases of zeta be exceuted in CI/CD, how to trigger it manually? cc @EricJoy2048 @Hisoka-X

Please track https://github.com/apache/incubator-seatunnel/pull/4038

**Comment by TyrantLucifer** [comment]
> > I didn't see any it cases of zeta be exceuted in CI/CD, how to trigger it manually? cc @EricJoy2048 @Hisoka-X
> 
> Please track #4038

OK, got it.

**Comment by liugddx** [comment]
> > I didn't see any it cases of zeta be exceuted in CI/CD, how to trigger it manually? cc @EricJoy2048 @Hisoka-X
> 
> Please track #4038

This PR is a Hazelcast feature. The test cases of zeta will cover this optimization. The problem with the last test was due to the first port being taken, and the problem exists now. We need to solve the problem that the port is occupied and the master cannot be obtained.

**Comment by liugddx** [comment]
image

File: config/hazelcast.yaml [hunk 1]
```diff
@@ -17,20 +17,35 @@
...
-  network:
-    rest-api:
-      enabled: true
+  advanced-network:
+    enabled: true
+    member-server-socket-endpoint-config:
+      port:
+        auto-increment: true
+        port-count: 100
+        port: 15801
```
Comment (Hisoka-X) [inline config/hazelcast.yaml:26]: Still use 5801 series port for member socket, not change it. Use 15801 for client maybe be better.

**Pull request closed without merging**